src/api-binder/report.ts

```diff
@@ -138,12 +129,21 @@ export async function startReporting() {
 		'appUpdatePollInterval',
 	])) as StateReportOpts;
 
-	let reportPending = false;
+	const withThrottle = pThrottle({
+		limit: Infinity,
```

Comment:
To report once every maxReportFrequency, shouldn't limit = 1?

Reply:
that makes much more sense than what I assumed it for was xD thanks